packages/console/src/pages/SignInExperience/index.tsx

```diff
@@ -72,20 +70,20 @@ const SignInExperience = () => {
           {!data && !error && <div>loading</div>}
           {error && <div>{`error occurred: ${error.body.message}`}</div>}
           {data && (
-            <form onSubmit={onSubmit}>
-              {tab === 'experience' && (
-                <BrandingForm register={register} control={control} watch={watch} />
-              )}
-              {tab === 'experience' && <TermsForm register={register} watch={watch} />}
-              <div className={detailsStyles.footer}>
-                <Button
-                  disabled={isSubmitting}
-                  type="primary"
-                  htmlType="submit"
-                  title="general.save_changes"
-                />
-              </div>
-            </form>
+            <FormProvider {...methods}>
+              <form onSubmit={onSubmit}>
+                {tab === 'experience' && <BrandingForm />}
+                {tab === 'experience' && <TermsForm />}
```

Comment:
These two can be merged in one condition

Reply:
updated